cluster-state-service/handler/reconcile/loader/translate.go

```diff
@@ -54,6 +53,10 @@ func ToTask(ecsTask ecs.Task) types.Task {
 		stoppedAt := ecsTask.StoppedAt.Format(timeLayout)
 		taskDetail.StoppedAt = stoppedAt
 	}
+	if ecsTask.Version == nil {
```

Comment:
Is this expected to ever happen? If yes, when? And do you think it would be better to ignore an unversioned response?